test/e2e/app-dir/index.test.ts

```diff
@@ -1194,6 +1194,27 @@ describe('app dir', () => {
             expect(el.attr('data-query')).toBe('query')
           })
         })
+
+        describe('useSelectedLayoutSegment', () => {
+          test.each`
+            path                                                           | outerLayout                                             | innerLayout
+            ${'/hooks/use-selected-layout-segment/first'}                  | ${['first']}                                            | ${[]}
```

Comment:
Could you add a test for calling `useSelectedLayoutSegments` in `page.js`, that should just return `[]`